.github/workflows/test.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           PKG_CONFIG_PATH: /usr/local/opt/openssl@3/lib/pkgconfig:/opt/homebrew/opt/openssl@3/lib/pkgconfig:/usr/local/opt/libffi/lib/pkgconfig:/opt/homebrew/opt/libffi/lib/pkgconfig
           NAT_BUILD_MODE: release
         run: |
-          export PATH="/opt/homebrew/opt/ruby@3.3/bin:$PATH"
```

Comment:
Hmmm, for a short period yesterday, the `/opt/homebrew/opt/ruby` path didn't exist. But I guess they fixed it. 🤷‍♂️

Reply:
The same path (without the explicit version) is used in an earlier debug step to print the ruby version, so this makes it more consistent as well.